xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp

```diff
@@ -607,8 +607,31 @@ void CLinuxRendererGLES::RenderUpdateVideo(bool clear, DWORD flags, DWORD alpha)
     CDVDVideoCodecIMXBuffer *buffer = m_buffers[m_iYV12RenderBuffer].IMXBuffer;
     if (buffer != NULL && buffer->IsValid())
     {
-      ManageDisplay();
-      g_IMXContext.SetBlitRects(m_sourceRect, m_destRect);
+      RENDER_STEREO_MODE stereo_mode = g_graphicsContext.GetStereoMode();
+      if (stereo_mode)
+        g_graphicsContext.SetStereoView(RENDER_STEREO_VIEW_LEFT);
+
+     ManageDisplay();
+
+     if (stereo_mode = g_graphicsContext.GetStereoMode())
+       g_graphicsContext.SetStereoView(RENDER_STEREO_VIEW_OFF);
+
+     CRect srcRect(m_sourceRect);
+     CRect dstRect(m_destRect);
+     switch (stereo_mode)
+     {
+       case RENDER_STEREO_MODE_SPLIT_HORIZONTAL:
+         // fix me
```

Comment:
What you should have is 0,0 1920,540 for a TAB and 0,0 960,1080 for a SBS.
In ManageDisplay, be sure you have
```
      // Those are flipped in y
      if (m_format == RENDER_FMT_CVBREF || m_format == RENDER_FMT_MEDIACODEC)
```
= No IMX.